### DISCUSSION TITLE
Would be great to connect to (Open)-Subsonic API in the future

### DISCUSSION THREAD

**Discussion by drloloto** [basharovV/musicat#24] [Ideas]
Would be great to extend the capabilities to use the Subsonic API to support already given infrastructure.

Website Open Subsonic: https://opensubsonic.netlify.app/
Typescript API for Subsonic : https://github.com/explodingcamera/subsonic-api
Awesome List with more info and providers: https://github.com/basings/selfhosted-music-overview?tab=readme-ov-file#client-overview

What is you intention and direction with the media player?

**Comment by basharovV** [comment]
Hi! I'm interested in developing a media server mode for the desktop app at some point in the future, as well as a client mode. Possibly also detaching the Svelte UI and offering a webapp with the client mode only. Subsonic could be a viable solution, but I need to look into it further. Thanks for the suggestion!
